src/enclave/enclave_signal.c

```diff
@@ -5,8 +5,8 @@
 
 #include <asm-generic/ucontext.h>
 
-#include <lkl_host.h>
 #include <lkl/setup.h>
+#include <lkl_host.h>
```

Comment:
To be precise, "a load" means exactly 1 line of diff? We've gone over this many times before - please merge the format-everything PR, otherwise you will be annoyed many, many times.

Reply:
There are several other changes.  We have, as you say, gone over it many times, yet it keeps happening.

Reply:
Yes, and it will keep happening if you don't enforce it. The "several other changes" amount to exactly 1 line of semantic change in this file.